bubblesub/cmd/set_views.py

```diff
@@ -0,0 +1,49 @@
+# bubblesub - ASS subtitle editor
+# Copyright (C) 2018 Marcin Kurczewski
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+
+from PyQt5 import QtWidgets
+
+from bubblesub.api import Api
+from bubblesub.api.cmd import BaseCommand
+
+
+class SetViewCommand(BaseCommand):
+    names = ["set-view"]
+    help_text = "Changes the GUI view."
+
+    async def run(self) -> None:
+        await self.api.gui.exec(self._run_with_gui)
+
+    async def _run_with_gui(self, main_window: QtWidgets.QMainWindow) -> None:
+        if self.args.view_name == "subs":
+            main_window.view.subs_view()
+        elif self.args.view_name == "audio":
+            main_window.view.audio_view()
+        elif self.args.view_name == "video":
+            main_window.view.video_view()
+        else:
+            main_window.view.full_view()
```

Comment:
I'd call one method with ViewState of user's choice rather than have four methods